.github/ISSUE_TEMPLATE/support-add-project-contributor.md

```diff
@@ -0,0 +1,21 @@
+---
+name: 'Support: Add me as Tractus-X project contributor'
+about: 'Add a user to our Eclipse Foundation project as an official contributor'
+title: 'New Tractus-X project contributor'
+labels: support
+assignees: ''
+---
+
+<!-- This issue template is used to request admission to the Eclipse Tractus-X project contributor group -->
+
+> __Before requesting the project contributor role, please make sure, that you:__
+>
+> - Make yourself familiar with the Eclipse project roles: https://eclipse-tractusx.github.io/docs/oss/contributor-committer
+> - Make ourself familiar with prerequisites, like a signed ECA: https://eclipse-tractusx.github.io/docs/oss/getting-started
+> - Check the committer liste of our project and higlight at least one of them on this issue, since ever committer
```

Comment:
Small typos: 
- liste vs. list
- _since ever committer_, should be _since every committer_